include/uv.h

```diff
@@ -431,10 +427,56 @@ UV_EXTERN int uv_send_buffer_size(uv_handle_t* handle, int* value);
 UV_EXTERN int uv_recv_buffer_size(uv_handle_t* handle, int* value);
 
 UV_EXTERN int uv_fileno(const uv_handle_t* handle, uv_os_fd_t* fd);
+UV_EXTERN int uv_dup(uv_os_fd_t fd, uv_os_fd_t* dupfd);
 
 UV_EXTERN uv_buf_t uv_buf_init(char* base, unsigned int len);
 
 
+#ifdef _MSC_VER
+# define INLINE __inline
+#else
+# define INLINE inline
+#endif
+
+INLINE static uv_os_fd_t uv_get_osfhandle(int fd) {
```

Comment:
I see! Can you please add a comment so someone doesn't f*ck this up in the future?